tox.ini

```diff
@@ -92,6 +92,38 @@ skip_install = true
 allowlist_externals =
   git
 
+
+[testenv:preview]
+description = preview the docs
+deps =
+  sphinx-autobuild
+  {[testenv:build-docs]deps}
+commands_pre =
+commands =
+  # Retrieve possibly missing commits:
+  -git fetch --unshallow
+  -git fetch --tags
+
+  # Build the html docs with sphinx-autobuild:
+  sphinx-autobuild \
+    -j auto \
+    -b html \
+    -n \
+    -W \
+    . \
+    --watch ../README.rst \
+    --watch ../CHANGELOG.md \
+    "{envdir}/docs_out"
+
+changedir = {toxinidir}/docs
+isolated_build = true
+passenv =
+  SSH_AUTH_SOCK
+skip_install = true
+allowlist_externals =
+  git
```

Comment:
Let's reuse all these values from the other env, thinking ahead about the maintainability..